pkg/logger/logger_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright © 2019 The Homeport Team
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package logger_test
+
+import (
+	"time"
+
+	. "github.com/homeport/disrupt-o-meter/pkg/logger"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Logger code test", func() {
+	Context("Testing default behaviour of logger instances", func() {
+		var (
+			channelProvider ChannelProvider
+			pipeline        *PipelineMock
+			cluster         Cluster
+			loggerFactory   Factory
+			logger          Logger
+		)
+
+		BeforeEach(func() {
+			channelProvider = NewChannelProvider(10)
+			pipeline = &PipelineMock{}
+			cluster = NewLoggerCluster(pipeline, channelProvider, time.Second*1)
+			loggerFactory = NewChanneledLoggerFactory(channelProvider)
+			logger = loggerFactory.NewChanneledLogger("test-logger")
+		})
+
+		It("Should have the correct name", func() {
```

Comment:
Nothing major, just that I never saw it with capital letter "S" before, just as "should".